webpack.common.js

```diff
@@ -4,6 +4,10 @@ const CopyWebpackPlugin = require('copy-webpack-plugin');
 const { ProvidePlugin } = require('webpack');
 
 module.exports = {
+  experiments: {
+    topLevelAwait: true,
+    asyncWebAssembly: true,
```

Comment:
`asyncWebAssembly` is not necessary if wasm is loaded as base64 string

Reply:
Fixed by 798ec81